src/components/GardensDashboard.js

```diff
@@ -1,124 +1,24 @@
-import React, { useCallback } from 'react'
-import { GU, shortenAddress, textStyle, useTheme } from '@1hive/1hive-ui'
-import { useHistory } from 'react-router'
-
-import { useGardens } from '../providers/Gardens'
-
-import defaultGardenLogo from '../assets/defaultGardenLogo.svg'
-import defaultTokenLogo from '../assets/defaultTokenLogo.svg'
+import React from 'react'
+import styled from 'styled-components'
+import GardensList from './GardensList'
+import LandingBanner from './LandingBanner'
+import { useNodeHeight } from '@hooks/useNodeHeight'
 
 function GardensDashboard() {
-  // TODO :  add loading component
-  const { gardens } = useGardens()
+  const [height, ref] = useNodeHeight()
```

Comment:
This hook tracks the landing Banner container's height so we can dynamically set the gardens list container margin.

This is because the banner is absolute positioned (since it's inside the `Layout` component which set's left and right margins resulting in the banner not having the full window width otherwise) so we need to place the gardens list container perfectly below the banner.